TestingLowerBounds/KullbackLeibler.lean

```diff
@@ -154,4 +154,29 @@ lemma kl_nonneg (μ ν : Measure α) [IsProbabilityMeasure μ] [IsProbabilityMea
 
 end kl_nonneg
 
+section Conditional
+
+variable {β : Type*} {mβ : MeasurableSpace β}
+
+open Classical in
+
+-- TODO: choose the right definition between the following two
+noncomputable
+def cond_kl (κ η : kernel α β) (μ : Measure α) : EReal :=
```

Comment:
The names of definitions should be camel case (and let's capitalize kl into KL rather that Kl):
```suggestion
def condKL (κ η : kernel α β) (μ : Measure α) : EReal :=
```